.maestro/testsV2/subFolder/links/DeepLinksGeneric.yml

```
@@ -0,0 +1,18 @@
+appId: app.passculture.webapp
+---
+- runFlow:
+    when:
+      true: ${MAESTRO_APP_INSTALLED == 'true'}
+    commands:
+      - stopApp
+
+- runFlow:
+    when:
+      true: ${MAESTRO_APP_INSTALLED == 'false'}
+    commands:
+      - stopApp: com.android.chrome
```

si on garde ce fichier, il faudrait réutiliser le flow générique qui fait ça 

```suggestion
- runFlow: ../Close.yml
```